mlp.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Optional, Tuple
+
+import torch
+from pytorch_lightning import LightningModule
+from torch import Tensor
+from torch.nn import BCELoss, Linear, Sigmoid
+from torch.optim import AdamW
+
+
+class LitLinear(LightningModule):
```

Comment:
Nit: I don't like including "Lit" or anything that references Lightning in our class name, because our IDE and the inheritance is going to show us this already anyway. Ultimately we don't care about using Lightning, and if they change their name, we don't want our code to look irrelevant either. Better to just name what the model does than to reference a framework name:

```suggestion
class SingleLinear(LightningModule):  # or SimpleLinear, or OneLinear, etc.
```

Reply:
Done